Project.toml

```diff
@@ -7,6 +7,7 @@ ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Richardson = "708f8203-808e-40c0-ba2d-98a6953ed40d"
 
```

Comment:
Need to bump the version.
Are we happy enough with the public API to call this v1?

Reply:
I'm not sure what's best. For now, I've changed the version to `0.11.0`. Let me know if you want to go for a major release instead.